src/App.jsx

```diff
@@ -1,44 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const visibleGoods = [...tabs];
+  const [activeTabId, onTabSelected] = useState(visibleGoods[0].id);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+  const currentTab =
+    visibleGoods.find(visibleGood => visibleGood.id === activeTabId) ||
+    visibleGoods[0];
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">Selected tab is {currentTab.title}</h1>
 
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
+      <div data-cy="TabsComponent">
+        <Tabs
+          goods={visibleGoods}
+          currentTab={currentTab}
+          set={id => {
+            onTabSelected(id);
+          }}
```

Comment:
The `Tabs` component expects the following props: `tabs`, `activeTabId`, and `onTabSelected` as per the task description. However, you are passing `goods`, `currentTab`, and `set`. Please update these prop names to match the expected ones.